src/ImageSharp.Web.Providers.AWS/IAWSS3BucketClientOptions.cs

```diff
@@ -1,13 +1,20 @@
 // Copyright (c) Six Labors.
 // Licensed under the Six Labors Split License.
 
+using Amazon.S3;
+
 namespace SixLabors.ImageSharp.Web;
 
 /// <summary>
 /// Provides a common interface for AWS S3 Bucket Client Options.
 /// </summary>
-internal interface IAWSS3BucketClientOptions
+public interface IAWSS3BucketClientOptions
```

Comment:
I had to make the internal interface public here, because it is used as a param of the S3ClientProvider.